src/test/java/net/oauth/jsontoken/JsonTokenParserTest.java

```diff
@@ -221,21 +221,17 @@ private boolean verifyTimeFrame(Instant issuedAt, Instant expiration) throws Exc
 
   private void deserializeExpectIllegalStateException(String tokenString) throws Exception {
     JsonTokenParser parser = new JsonTokenParser(clock, locators, new IgnoreAudience());
-    try {
-      parser.deserialize(tokenString);
-      fail("Expected IllegalStateException");
-    } catch (IllegalStateException e) {
-      // expected
-    }
+    assertThrows(
+        IllegalStateException.class,
+        () -> parser.deserialize(tokenString)
+    );
   }
 
   private void deserializeExpectJsonParseException(String tokenString) throws Exception {
     JsonTokenParser parser = new JsonTokenParser(clock, locators, new IgnoreAudience());
-    try {
-      parser.deserialize(tokenString);
-      fail("Expected JsonParseException");
-    } catch (JsonParseException e) {
-      // expected
-    }
+    assertThrows(
+        JsonParseException.class,
+        () -> parser.deserialize(tokenString)
+    );
   }
```

Comment:
nit: Can we just write these 2 statements in-line instead of using helper methods? It's useful to separate the "arrange" (test setup) part from the "act" and "assert" steps for readability (see the internal link I shared with you separately about unit testing practices).

Reply:
Moved